sklearn/cluster/_kmeans.py

```diff
@@ -1252,156 +1345,60 @@ def _mini_batch_step(X, sample_weight, x_squared_norms, centers, weight_sums,
     verbose : bool, default=False
         Controls the verbosity.
 
+    n_threads : int, default=1
+        The number of OpenMP threads to use for the computation.
+
     Returns
     -------
     inertia : float
         Sum of squared distances of samples to their closest cluster center.
```

Comment:
I think it would help to make it explicit that this is the inertia before observed on the batch before updating the cluster centers.